src/app/work-item/work-item-detail/work-item-detail.component.ts

```diff
@@ -324,7 +324,7 @@ export class WorkItemDetailComponent implements OnInit, AfterViewInit, OnDestroy
   getAllUsers(): Observable<any> {
     return Observable.combineLatest(
       this.userService.getUser(),
-      this.userService.getAllUsers()
+      this.workItemService.getCollaborators()
```

Comment:
You can actually update that file to use **.relationships.collaborators.links.related** in line 24